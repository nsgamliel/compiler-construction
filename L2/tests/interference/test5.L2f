(:myF
0

%variable3FXEWM9YA6 <- rbx

  rbx <- 5
  %myVar1 <- rbx

rbx <- %variable3FXEWM9YA6
  return
)
