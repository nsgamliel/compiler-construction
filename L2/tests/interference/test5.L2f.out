%myVar1 r12 r13 r14 r15 rax rbp %variable3FXEWM9YA6
r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r11 r10 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r12 %myVar1 r10 r11 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %variable3FXEWM9YA6
r13 %myVar1 r10 r11 r12 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %variable3FXEWM9YA6
r14 %myVar1 r10 r11 r12 r13 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %variable3FXEWM9YA6
r15 %myVar1 r10 r11 r12 r13 r14 r8 r9 rax rbp rbx rcx rdi rdx rsi %variable3FXEWM9YA6
r8 r10 r11 r12 r13 r14 r15 r9 rax rbp rbx rcx rdi rdx rsi
r9 r10 r11 r12 r13 r14 r15 r8 rax rbp rbx rcx rdi rdx rsi
rax %myVar1 r10 r11 r12 r13 r14 r15 r8 r9 rbp rbx rcx rdi rdx rsi %variable3FXEWM9YA6
rbp %myVar1 r10 r11 r12 r13 r14 r15 r8 r9 rax rbx rcx rdi rdx rsi %variable3FXEWM9YA6
rbx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rcx rdi rdx rsi %variable3FXEWM9YA6
rcx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rdi rdx rsi
rdi r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdx rsi
rdx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rsi
rsi r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx
%variable3FXEWM9YA6 %myVar1 r12 r13 r14 r15 rax rbp rbx
